py/newsletter.py

```diff
@@ -20,14 +24,14 @@
 # create the document body
 with doc:
     # enclose all of the elements within div elements (currently set to ef-styling)
-    with div(cls='ef-wrapper').add(div(cls='ef-container')).add(div(cls='ef-table ef-outer')):
+    with div(cls='wrapper').add(div(cls='container')).add(table(cls='table outer')):
         _head()
-        _greetings()
-        _bodylineleft()
-        _bodylineright()
-        _bodynoline()
-        _foot()
+        _greetings(week)
+        # _bodylineleft()
```

Comment:
let's remove the massive commenting out of the code, this is just a change that will need to be reverted later.

Reply:
Removed comments